evaluation/aider_bench/scripts/summarize_results.py

```diff
@@ -1,61 +1,25 @@
-import json
-import os
-import sys
+import argparse
```

Comment:
I didn't fully follow the changes in this file, and they seem orthogonal to the main point of this commit?

Reply:
yep - i can break this into a separate PR (or just remove this changes of this file completely)